### PR TITLE
User settings

### PR DESCRIPTION
**Pull request by mmamongy**

### CODE REVIEW

File: src/app/settings/payments/payments.component.ts [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+  paramsObject: any;
+  price: any;
+  packageId: any;
+  
 
-  constructor() { }
+  constructor( private activeRoute : ActivatedRoute) { 
+    this.packageId = this.activeRoute.snapshot.paramMap.get("pid") ;
+    this.price = this.activeRoute.snapshot.paramMap.get("price");
+    console.log(this.packageId , this.price) ;
```
Comment (Aychihuahuausa) [inline src/app/settings/payments/payments.component.ts:18]: remove console log

File: src/app/settings/payments/payments.component.ts [hunk 2]
```diff
@@ -1,13 +1,25 @@
 import { Component, OnInit } from '@angular/core';
+import {ActivatedRoute  } from '@angular/router' ;
 
 @Component({
   selector: 'app-payments',
   templateUrl: './payments.component.html',
   styleUrls: ['./payments.component.scss'],
 })
 export class PaymentsComponent implements OnInit {
+  paramsObject: any;
```
Comment (Aychihuahuausa) [inline src/app/settings/payments/payments.component.ts:10]: define types for each prop

File: src/app/settings/paypal-payment/paypal-payment.component.ts [hunk 3]
```diff
@@ -8,11 +8,19 @@ import { PayPal, PayPalPayment, PayPalConfiguration } from '@ionic-native/paypal
   styleUrls: ["./paypal-payment.component.scss"],
 })
 export class PaypalPaymentComponent implements OnInit {
-  ngOnInit() {}
-  paymentAmount: string = '3.33';
+  packageId: any;
+  price: any;
```
Comment (Aychihuahuausa) [inline src/app/settings/paypal-payment/paypal-payment.component.ts:12]: define type for each prop

File: src/app/settings/payments/payments.component.ts (hunk 2 above)
Comment (mmamongy) [reply src/app/settings/payments/payments.component.ts:10]: fixed

File: src/app/settings/paypal-payment/paypal-payment.component.ts (hunk 3 above)
Comment (mmamongy) [reply src/app/settings/paypal-payment/paypal-payment.component.ts]: fixed

File: src/app/settings/barkbucks/barkbucks.component.html [hunk 4]
```diff
@@ -10,7 +10,10 @@
 </ion-header>
 <div>
   <ion-list>
-    <ion-item *ngFor="let i of barkbucks_list">
+    <ion-item [routerLink]="['/settings/payments', 1 , 0.5]">
+      <ion-label>Test with 50 Cents</ion-label>
```
Comment (heshambadawy) [inline src/app/settings/barkbucks/barkbucks.component.html:14]: Should we merge that test fixed values , What's the Purpose ?

Comment (Aychihuahuausa) [reply src/app/settings/barkbucks/barkbucks.component.html:14]: this is for test data

**Pull request merged**